src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1212,6 +1210,26 @@ namespace winrt::TerminalApp::implementation
                                                                                         TerminalSettings settings,
                                                                                         const bool inheritCursor)
     {
+        // The only way to create string references to literals in WinRT is through std::optional. Fun!
+        static std::optional<winrt::param::hstring> textMeasurement;
+        static const auto textMeasurementInit = [&]() {
```

Comment:
Couple thoughts. `param` doesn't get us much here - right? I don't think we're using its magic translation abilities...

Why doesn't `static winrt::hstring textMeasurement = [&](){ case ... return L"foo"; }()` work?

Reply:
I can use `winrt::hstring` here. I just got a little annoyed about not being able to reference a static string when using cppwinrt when I wrote this. 😅

(You can't return a `winrt::param::hstring` because its move constructor is explicitly deleted.)